Bomberfrau/src/main/java/uni/bombenstimmung/de/game/PlayerHandler.java

```diff
@@ -176,7 +219,7 @@ public static void drawPlayers(Graphics g) {
      * @param keyCode	Tasten-Code in Integer-Form
      */
     public static void handleKeyEventPressed(int keyCode) {
-	if (clientPlayer.getDead() == false && Game.getGameOver() == false) {
+	if (clientPlayer.getDead() == false && Game.getGameOver() == false && movable == true) {
```

Comment:
Eventuell 
`getDead()` in Player.java umbennen zu `isDead()`
boolean werden i.d.R. mit `is` abgefragt.